sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -261,6 +261,17 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val HADOOPFSRELATION_SIZE_FACTOR = buildConf(
```

Comment:
How about `DISK_TO_MEMORY_SIZE_FACTOR`? IMHO the current name doesn't describe the purpose clearly.